deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -15,6 +15,8 @@ featureGates:
 
 priorityClassName: ""
 
+cleanupLabels: true
```

Comment:
nitpick about the naming. It cleans up other stuff too (i.e. annotations, extended resources and taints), so I'd suggest something like `postDeleteCleanup` or smth. WDYT @alekseysen